.circleci/config.yml

```diff
@@ -19,6 +31,17 @@ jobs:
             - node/install-packages:
                   pkg-manager: pnpm
             - run:
-                  command: pnpm build
```

Comment:
Since you remove build step from "test" job, which I think is a good thing, shall you add it to it's separate job that precedes running tests?

Reply:
It's triggered automatically through task dependency in turborepo config